include/envoy/http/filter.h

```diff
@@ -292,6 +292,12 @@ class FilterChainFactoryCallbacks {
    * @param filter supplies the filter to add.
    */
   virtual void addStreamFilter(Http::StreamFilterPtr filter) PURE;
+
+  /**
+   * Add an access log handler that is called after response is send.
```

Comment:
"Add an access log handler that is called when the stream is destroyed." (A response may not have been sent)